cookiecutter/{{cookiecutter.project_name}}/examples/CMakeLists.txt

```diff
@@ -1,5 +1,31 @@
 # SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 
+cmake_minimum_required(VERSION 3.25...4.1)
+
+project(
+    beman.{{cookiecutter.project_name}}.examples
+    DESCRIPTION "Examaples for Beman implementation of {{cookiecutter.project_name}}"
+    LANGUAGES CXX
+    VERSION 2.1.1
+)
+
+if(PROJECT_IS_TOP_LEVEL)
```

Comment:
The example code should not be used for testing. This code serves no purpose for a user who is copy-pasting the example into their own source directory. It serves only to confuse them.

The test should be a five line project that calls `cmake_minimum_required()` -> `project()` -> `find_package(REQUIRED)` -> `if(NOT TARGET)`. It should not be an example. It should exist solely for the purpose of testing the imports.